aws/redshift/tables/regional_partner_stats_csp_csd.sql

```diff
@@ -0,0 +1,130 @@
+--CHANGES
+-- addition of school years beyond 2017
+-- removed 'location'
+
+-- CHANGED DEPENDENCIES
+-- before using this code to update analysis.regional_partner_stats_csp_csd, certain other tables need to have their names changed, and their references updated here
+-- similarly, the tables analysis.teacher_most_progress and analysis.student_activity need to be updated and then their references in this file replaced with the originals
+-- these two also need to be updated on Github
+
+-- NOTES ON HOW TO UPDATE 
+-- after updating at the dependencies listed above and noted with 'PUBLIC' in the code....
+-- replace all instances of 'public.regional_partner_stats_csp_csd_test' with 'analysis.regional_partner_stats_csp_csd' and re-run table creation
+-- this table will be generated daily from code hosted on github, so need to talk to Ben about how to update that process
+
+;
+drop table if exists analysis.regional_partner_stats_csp_csd;
+
+create table analysis.regional_partner_stats_csp_csd 
+AS
+with completed as
+(
+  select 
+    studio_person_id,
+    case when cc.course_name = 'csp' then 'CS Principles'
+         when cc.course_name in ('csd','csd_half') then 'CS Discoveries'
+    end as course,
+    school_year
+  from csp_csd_completed_teachers cc
+  join users u on u.id = cc.user_id
+), 
+started as
+(
+  select 
+    studio_person_id,
+    case when cc.course_name = 'csp' then 'CS Principles'
+         when cc.course_name in ('csd','csd_half') then 'CS Discoveries'
+    end as course,
+    school_year
+  from csp_csd_started_teachers cc
+  join users u on u.id = cc.user_id
+),
+pd_enrollments_with_year as
```

Comment:
Aggregate to studio person level to avoid duplicating rows for teachers with multiple accounts